packages/cli/src/commands/create-app/createApp.ts

```diff
@@ -88,25 +88,39 @@ export async function moveApp(
 
 async function addPackageResolutions(appDir: string) {
   const pkgJsonPath = resolvePath(appDir, 'package.json');
-  const packageFileContent = await fs.readFile(pkgJsonPath, 'utf-8');
-  const packageFileJson = JSON.parse(packageFileContent);
-
-  packageFileJson.resolutions = packageFileJson.resolutions || {};
-
-  const packages = ['cli', 'core', 'test-utils', 'test-utils-core', 'theme'];
-
-  for (const pkg of packages) {
-    await Task.forItem('adding', `${pkg} link to package.json`, async () => {
-      const pkgPath = paths.resolveOwnRoot('packages', pkg);
-      packageFileJson.resolutions[`@backstage/${pkg}`] = `file:${pkgPath}`;
-      const newContents = `${JSON.stringify(packageFileJson, null, 2)}\n`;
+  const pkgJson = await fs.readJson(pkgJsonPath);
+
+  pkgJson.resolutions = pkgJson.resolutions || {};
+  pkgJson.dependencies = pkgJson.dependencies || {};
+
+  const depNames = [
```

Comment:
At some point we talked about reading these programmatically right? Maybe next time ™

Reply:
Been there and went back x) Didn't wanna get all the plugins installed when it wasn't needed. Thinking about it again it might be worth it though.